integration/src/test/java/org/apache/iotdb/db/integration/IoTDBRestartIT.java

```diff
@@ -434,7 +434,6 @@ public void testRestartCompaction()
 
     try {
       CompactionTaskManager.getInstance().waitAllCompactionFinish();
-      Thread.sleep(10000);
```

Comment:
Why do you remove it? I remember the tests may fail if not sleep here...

Reply:
In the `waitAllCompactionFinish()` it will wait until `taskExecutionPool.getActiveCount() == 0 && taskExecutionPool.getQueue().size() == 0`. So I don't know what the `sleep()` are waiting for?

Reply:
@THUMarkLau This is relatived to your work. Could you explain the design of this test case :)

Reply:
`testRestartCompaction` can be removed in 0.13, because we redesigned and implemented the Compaction module. The amount of data inserted in this test does not trigger a compaction. There are more detailed tests of restarting a Compaction than this one.

Reply:
Thanks! I have deleted this test case in the current PR